telemetry-all/build.gradle.kts

```diff
@@ -1,44 +1,44 @@
-private object Versions {
-        const val slf4j = "1.7.30"
-        const val gson = "2.8.6"
-}
-
 plugins {
     java
     id("java-library")
+    id( "org.ysb33r.java.modulehelper")
```

Comment:
I am hesitant to have our production build path requiring use of a nonstandard, external 3rd party plugin (even if @ysb33r did create it).  Perhaps we can just include the plugin in this project instead?

Reply:
What do you define as standard vs non-standard?

Reply:
To follow up on this - we've agreed that the ModuleHelper plugin will migrate into the New Relic org in the short-medium term. So I think it's fine to depend upon it in its current location for now - but will clean it up once the migration has happened.